_includes/footer.html

```diff
@@ -43,18 +48,53 @@
   integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
   crossorigin="anonymous"
 ></script>
+<script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/vue"></script>
 
 <script>
-  function on() {
-    document.getElementById("overlay").style.display = "block";
-  }
+    function on() {
+      document.getElementById("overlay").style.display = "block";
+    }
 
-  function off() {
-    document.getElementById("overlay").style.display = "none";
-  }
+    function off() {
+      document.getElementById("overlay").style.display = "none";
+    }
 
-  $(function() {
-    $('img').addClass('img-fluid');
-  })
+    $(function() {
+      $('img').addClass('img-fluid');
+    })
 
+    var cantstopcbus = new Vue({
+      el: '#carousel2',
+      delimiters:['{$', '$}'],
+      data: {
+        projects: []
+      },
+      mounted() {
+        axios.get("https://wduc7ys73l.execute-api.us-east-1.amazonaws.com/dev/projects")
+          .then((response) => {
+            let shuffle = (array) => {
+              for(var i = array.length - 1; i> 0; i--) {
+                const j = Math.floor(Math.random() * i)
+                const temp = array[i]
+                array[i] = array[j]
+                array[j] = temp
+              }
+              return array
+            }
```

Comment:
Should there be a check that response was ok? I don't remember if `axios` does a successful return on a 404.

Reply:
good question. i'll check.